src/common/history.c

```diff
@@ -925,20 +925,28 @@ char *dt_history_get_name_label(const char *name,
                                 const char *label,
                                 const gboolean markup)
 {
+  char *nesc = g_markup_printf_escaped("%s", name);
+  char *lesc = g_markup_printf_escaped("%s", label);
+  char *result = NULL;
+
   if(!label
      || strlen(label) == 0
      || strcmp(label, "0") == 0)
   {
-    return g_strdup_printf("%s", name);
+    result = nesc;
   }
   else
   {
-    return g_strdup_printf("%s • %s%s%s",
-                           name,
-                           markup ? "<small>" : "",
-                           label,
-                           markup ? "</small>" : "");
+    result = g_strdup_printf("%s • %s%s%s",
+                             nesc,
+                             markup ? "<small>" : "",
+                             lesc,
+                             markup ? "</small>" : "");
```

Comment:
```
result = markup ? g_strdup_printf("%s • %s", name, label)
                : g_markup_printf_escaped("%s • <small>%s<small>", name, label);
```

Reply:
This is reversed :) But you can't escape here as it will print `<small>` instead of applying it.

Reply:
Github emailed me your reply but I can't see it here. Obviously this is reversed :-) Sorry. Also doesn't take care of `!label`. But was meant as a quick hint. `g_markup_printf_escaped` is different from `g_markup_escape_text` in that it _doesn't_ escape the format text.

Reply:
Yes, my reply was partly ok and partly wong, so I deleted it since I had to leave for a long bike ride and was already short on time :)